src/vs/workbench/contrib/audioCues/browser/audioCueService.ts

```diff
@@ -165,6 +165,12 @@ export class AudioCue {
 		settingsKey: 'audioCues.debuggerStoppedOnBreakpoint',
 	});
 
+	public static readonly noInlayHints = AudioCue.register({
+		name: localize('audioClues.noInlayHints', 'No Inlay Hints available for the current line'),
+		sound: Sound.error,
+		settingsKey: 'audioClues.noInlayHints'
```

Comment:
It is not `audioClues` it is `audioCues` :)

Reply:
fixed in main